package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "inventoryflo-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "axios": "^0.19.2",
+    "bootstrap": "^4.4.1",
+    "jquery": "^3.5.1",
+    "popper.js": "^1.16.1",
```

Comment:
I think we can try other UI components like ant design. bootstrap will make the code dirty and plenty